### PR TITLE
0.8.1

### PR DESCRIPTION
**Pull request by MoOx**

### CODE REVIEW

**Comment by thangngoc89** [comment]
:+1: lgtm

File: src/builder/server.js [hunk 1]
```diff
@@ -125,56 +115,76 @@ export default (webpackConfig, options = {}) => {
...
-
-        if (!firstRun) {
-          cleanNodeCache(config.cwd)
-        }
-        firstRun = false
 
-        urlAsHtml(item.__url, {
-          exports: options.exports,
-          store: options.store,
+      // try 404.html is there is any
```
Comment (thangngoc89) [inline src/builder/server.js]: * if there is any

Comment (MoOx) [inline src/builder/server.js]: good catch haha

**Comment by thangngoc89** [comment]
Found that invalid checksum on 404 page. but in my case this happens on both firefox and chrome

**Comment by MoOx** [comment]
Will open an issue for that.

**Pull request merged**